pkg/uninstall/uninstall.go

```diff
@@ -80,6 +82,14 @@ func baseResources(ctx context.Context, c kclient.Client) (resources []kclient.O
 		}
 	}
 
+	controllerLease := &coordinationv1.Lease{}
+	if err := c.Get(ctx, kclient.ObjectKey{Namespace: "kube-system", Name: "acorn-controller"}, controllerLease); !apierror.IsNotFound(err) && err != nil {
```

Comment:
Does the lease live in acorn-system or kube-system?

Reply:
It lives in the `kube-system` namespace.